### PR TITLE
Close 'old' enquiries

### PR DESCRIPTION
**Pull request by pezholio**
For mysociety/fixmystreetforcouncils#51

This will mark all reports with the last update in 2015 as closed, and email the users with information on what problems have been closed, and the option to reopen them. Anything still open with a last update of 2014 or earlier gets closed with no email.

This could do with a code review, but would also be good to test against a corpus of real data before pointing it at live.

### CODE REVIEW

**Comment by pezholio** [comment]
I think this is ready for review now :+1:

File: perllib/FixMyStreet/DB/Result/Problem.pm [hunk 1]
```diff
@@ -511,6 +513,30 @@ sub admin_url {
     return $cobrand->admin_base_url . '/report_edit/' . $self->id;
 }
 
+=head2 tokenised_url
+
+Return a url for this problem report that logs a user in
+
+=cut
+
+sub tokenised_url {
```
Comment (pezholio) [inline perllib/FixMyStreet/DB/Result/Problem.pm]: I get what you're saying, but the issue there is that at the moment we've got dynamic attributes (namely `$user` and `$params`) that get passed in too. Is there a way round that?

File: templates/email/default/archive.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  <p style="[% p_style %]">
+    FixMyStreet is being updated in Oxfordshire to
+    improve how problems get fixed.
+  </p>
+  <p style="[% p_style %]">
+    As part of these updates, we are closing old reports that appear to be
+    resolved but remain open in the system.
+  </p>
+  <p style="[% p_style %]">
+    We noticed that you have [% report_count %] old reports on the system,
```
Comment (pezholio) [inline templates/email/default/archive.html]: Is there a Perl / Catalyst / tt2 way I can get around this?

File: templates/email/default/archive.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
+[%
```
Comment (pezholio) [inline templates/email/default/archive.html]: Hmmm... Moving the templates gives me `Template processing error: file error - archive.txt: not found` 😕

Comment (pezholio) [inline templates/email/default/archive.html]: Looking at the code `$cobrand->path_to_email_templates($lang_code)` seems to be returning `/templates/email/default` for some reason

Comment (pezholio) [inline templates/email/default/archive.html]: No wait, I got this. `FixMyStreet::override_config` is the key

File: templates/email/default/archive.html (hunk 2 above)
Comment (pezholio) [inline templates/email/default/archive.html]: Ah, `[% nget('reports', 'report', report_count) %]` seems to do the trick 👍

**Comment by Aliyadaulah** [comment]
hi please contact me at(aliyadaulah@gmail.com) for more about me from Aliya

**Pull request merged**